articles/140_topic_and_service_name_mapping.md

```diff
@@ -198,51 +198,17 @@ Any topic or service name that contains any tokens (either namespaces or a topic
 
 ## Mapping of ROS 2 Topic and Service Names to DDS Concepts
 
-The ROS topic and service name constraints allow more types of characters than the DDS topic names because ROS additionally allows the forward slash (`/`), the tilde (`~`), and the balanced curly braces (`{}`).
+The ROS topic and service name constraints allow more types of characters than the DDS topic names because ROS additionally allows the, the tilde (`~`), and the balanced curly braces (`{}`).
```

Comment:
is the `the ,` a typo or do you want to highlight the comma specifically?

Reply:
Its a typo